packages/mobile/src/escrow/saga.ts

```diff
@@ -86,7 +86,7 @@ function* registerStandbyTransaction(id: string, value: string, address: string)
       id,
       type: TransactionTypes.ESCROW_SENT,
       status: TransactionStatus.Pending,
-      value,
+      value: +value,
```

Comment:
Note that storing StandbyTransaction as `number` instead of `string does not impact precision, as the confirmed transaction value is stored as a `number` anyway. Did not want to modify graphQL to change both to strings so as not to overlap with @jeanregisser 's local currency work